whois-api/src/main/java/net/ripe/db/whois/api/rest/marshal/AbstractStreamingMarshalCommon.java

```diff
@@ -0,0 +1,63 @@
+package net.ripe.db.whois.api.rest.marshal;
+
+import net.ripe.db.whois.api.rest.RestServiceHelper;
+import net.ripe.db.whois.common.Message;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+public class AbstractStreamingMarshalCommon implements  StreamingMarshal{
```

Comment:
Also consider combining this class with `StreamingMarshal` to simplify the hierarchy (i.e. can they be a single class?)

Reply:
Done, Thank you! I did not realised